cookbook/frontend/index.rst

```diff
@@ -0,0 +1,7 @@
+Form
```

Comment:
this will be done by my PR and can be removed from this PR, to avoid merge conflicts